### PR TITLE
Standalone PBBA button + deep linking logic

### PR DESCRIPTION
**Pull request by mpetrenco**
### Changelog:
- Added `invokePayByBankApp` method to the Judo SDK;
- Added `JudoPBBAButton` branded button to the Judo SDK;
- Added fully configured PBBA button to the sample app;
- Added deep linking logic for PayByBankApp to the RN sample app;

### Notes:
- `TouchableOpacity` is buggy. The element that works for Android doesn't work for iOS and vice-versa. Had to use two separate packages as a workaround;
- For some reason, on Android, tapping the button might result in a **401** response on the `sale` request, saying that the request is malformed (**see demo for example**).

### Demo:
https://drive.google.com/file/d/1QgvPTG1ImW1YNRBetQHUSNTqWBD3LTgJ/view?usp=sharing
https://drive.google.com/file/d/13QzIoVxQLd9cfwTugTtNM8K1ocrR26b1/view?usp=sharing

### CODE REVIEW

File: sample-app/screens/PayByBankApp/PayByBankApp.tsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+import { TouchableOpacity as AndroidTouchableOpacity } from 'react-native-gesture-handler'
+
+export default class PayByBankApp extends Component {
+  constructor(props: any) {
+    super(props)
+    this.invokePayByBankApp = this.invokePayByBankApp.bind(this)
+  }
+
+  async invokePayByBankApp() {
+    const token = this.props.route.params.token
```
Comment (stpopa) [inline sample-app/screens/PayByBankApp/PayByBankApp.tsx:20]: const {token, secret, configuration} = this.props.route.params

File: sample-app/screens/Home/Home.tsx [hunk 2]
```diff
@@ -33,23 +34,84 @@ export default class Home extends Component {
...
+    const url = await Linking.getInitialURL()
+    if (url) {
+      this.updateDeepLinkURL(url, () => {
+        this.handlePBBATransaction()
+      })
+    }
+  }
+
+  updateDeepLinkURL(url: string | null, callback: Function) {
+    const mockURL = url + '&orderId=BTO-cUOLVpSbQ1Ce9M3a3Mc7_w'
```
Comment (stpopa) [inline sample-app/screens/Home/Home.tsx:71]: is this still needed?

File: sample-app/ios/sample-app.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -649,6 +649,7 @@
 						TestTargetID = 13B07F861A680F5B00A75B9A;
 					};
 					13B07F861A680F5B00A75B9A = {
+						DevelopmentTeam = 95V535S6TQ;
```
Comment (stpopa) [inline sample-app/ios/sample-app.xcodeproj/project.pbxproj:652]: can we revert these please?

File: sample-app/ios/Podfile.lock [hunk 4]
```diff
@@ -1,6 +1,6 @@
 PODS:
   - boost-for-react-native (1.63.0)
-  - CocoaDebug (1.2.8)
+  - CocoaDebug (1.2.7)
```
Comment (stpopa) [inline sample-app/ios/Podfile.lock:3]: let's keep the newer version

File: ios/Classes/Views/RNPBBAButtonManager.m [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+
+@interface RNPBBAButtonManager: RCTViewManager
+@end
+
+@implementation RNPBBAButtonManager
+
+static double const kPBBAButtonWidth = 310.0f;
+static double const kPBBAButtonHeight = 50.0f;
+
+
```
Comment (stpopa) [inline ios/Classes/Views/RNPBBAButtonManager.m:36]: extraline

File: sample-app/screens/Home/Home.tsx (hunk 2 above)
Comment (mpetrenco) [reply sample-app/screens/Home/Home.tsx:71]: @stpopa There's still no **orderId** in the URL.

**Pull request merged**